src/util/apiUtils.js

```diff
@@ -185,32 +187,44 @@ export const parseApiResponse = requestUrl => ([response, body]) => {
  */
 export const buildRequestArgs = externalRequestOpts =>
 	({ endpoint, params, flags }) => {
-
-		// cheap, brute-force object clone, acceptable for serializable object
-		const externalRequestOptsQuery = JSON.parse(JSON.stringify(externalRequestOpts));
-
-		externalRequestOptsQuery.url = encodeURI(`/${endpoint}`);
-		externalRequestOptsQuery.jar = createCookieJar(externalRequestOptsQuery.url);
+		const dataParams = querystring.stringify(params);
+		const headers = { ...externalRequestOpts.headers };
+		let url = encodeURI(`/${endpoint}`);
+		let body;
+		const jar = createCookieJar(url);
 
 		if (flags) {
-			externalRequestOptsQuery.headers['X-Meetup-Request-Flags'] = flags.join(',');
+			headers['X-Meetup-Request-Flags'] = flags.join(',');
 		}
 
-		const dataParams = querystring.stringify(params);
-
-		switch (externalRequestOptsQuery.method) {
-		case 'get':
-		case 'delete':
-			externalRequestOptsQuery.url += `?${dataParams}`;
-			externalRequestOptsQuery.headers['X-Meta-Photo-Host'] = 'secure';
-			break;
+		switch (externalRequestOpts.method) {
 		case 'post':
```

Comment:
that can be added in the PR that supports `PATCH`: https://github.com/meetup/meetup-web-platform/pull/223